boards/arm/atsame54_xpro/atsame54_xpro.dts

```diff
@@ -104,4 +104,15 @@ zephyr_udc0: &usb0 {
 &gmac {
 	status = "okay";
 	zephyr,random-mac-address;
+
+	phy: phy {
+		compatible = "phy,mii";
```

Comment:
This is failing CI because `phy` is not a device vendor. The failure is expected. The compatible should be changed.

From the PR description, this is an ethernet PHY, right?

Then please a look at this upstream Linux binding for what to do instead: https://github.com/torvalds/linux/blob/master/Documentation/devicetree/bindings/net/ethernet-phy.yaml